src/DevBetterWeb.Web/Pages/Admin/User.cshtml.cs

```diff
@@ -20,20 +21,23 @@ public class UserModel : PageModel
         private readonly UserManager<ApplicationUser> _userManager;
         private readonly RoleManager<IdentityRole> _roleManager;
         private readonly IUserRoleMembershipService _userRoleMembershipService;
+        private readonly AppDbContext _context;
```

Comment:
We don't use anything from Infrastructure in Web. Instead we would use an abstraction, like `IRepository`.

If you see `Infrastructure` in the list of using statements for a page, you know there is a problem.

Reply:
changes made